### DISCUSSION TITLE
Install react-native-view-shot to use addViewShot

### DISCUSSION THREAD

**Discussion by quanghoath** [tr3v3r/react-native-esc-pos-printer#180] [General]
I installed react-native-view-shot library but it still gives error
Please help me!

**Comment by tr3v3r** [comment]
> I installed react-native-view-shot library but it still gives error
> 
> Please help me!

Need details

**Reply by quanghoath** [reply]
![Simulator Screenshot - iPhone 15 - 2024-08-30 at 10 23 59](https://github.com/user-attachments/assets/f82ad8eb-8686-444e-871c-a1a3c330e304)
<img width="563" alt="Screenshot 2024-08-30 at 10 25 16" src="https://github.com/user-attachments/assets/ed33fce1-b00b-4926-9c63-a6a0acba3b6c">

i used addViewShot but print empty page?

**Reply by quanghoath** [reply]
> > I installed react-native-view-shot library but it still gives error
> > Please help me!
> 
> Need details

 WARN  Calling [RCTConvert UIImage:] on a background thread is not recommended

**Reply by imamnghr** [reply]
there is answer for this?
